commit 8518a9e2b013bdb087b535f4fc96ffbe13dc960f
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Fri Oct 1 14:05:21 2010 -0700

    staging: brcm80211: Remove unnecessary assertions.
    
    Remove assertions on the size of several structures.  These structures are
    never used anywhere.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/sys/wlc_ampdu.c
+++ b/drivers/staging/brcm80211/sys/wlc_ampdu.c
@@ -175,12 +175,6 @@ ampdu_info_t *BCMATTACHFN(wlc_ampdu_atta
 	int i;
 
 	/* some code depends on packed structures */
-	ASSERT(sizeof(struct dot11_bar) == DOT11_BAR_LEN);
-	ASSERT(sizeof(struct dot11_ba) == DOT11_BA_LEN + DOT11_BA_BITMAP_LEN);
-	ASSERT(sizeof(struct dot11_ctl_header) == DOT11_CTL_HDR_LEN);
-	ASSERT(sizeof(struct dot11_addba_req) == DOT11_ADDBA_REQ_LEN);
-	ASSERT(sizeof(struct dot11_addba_resp) == DOT11_ADDBA_RESP_LEN);
-	ASSERT(sizeof(struct dot11_delba) == DOT11_DELBA_LEN);
 	ASSERT(DOT11_MAXNUMFRAGS == NBITS(uint16));
 	ASSERT(ISPOWEROF2(AMPDU_TX_BA_MAX_WSIZE));
 	ASSERT(ISPOWEROF2(AMPDU_RX_BA_MAX_WSIZE));
